tools/autograd/templates/Functions.cpp

```diff
@@ -886,6 +886,41 @@ Tensor log_softmax_double_backward(const Tensor & grad, const Tensor & grad_outp
   return z * grad_output.sum(dim, true) * ((grad * z).sum(dim, true) - grad);
 }
 
+Tensor binary_cross_entropy_double_backward(const Tensor & grad_output, const Tensor & grad, const Tensor & input, const Tensor & target, const Tensor& weight, int64_t reduction) {
+  auto eps = 1e-12;
+  auto num = input * input - 2 * input * target + target;
+  auto denom = (input + eps) * (input + eps) * (1 - input + eps) * (1 - input + eps);
+  auto output = num/denom;
```

Comment:
nit: try to be consistent about spacing around operators.  Most of the time here you have spaces around the operators (which matches the usual convention, but here you don't.